simtools/db_handler.py

```diff
@@ -182,21 +180,32 @@ def getModelParameters(
 
     def exportFileDB(self, dbName, dest, fileName):
         """
-        Get a file from the DB and write it to disk.
+        Get file from the DB and write to disk.
 
         Parameters
         ----------
         dbName: str
             Name of the DB to search in.
         dest: str or Path
             Location where to write the file to.
-        fileName: str
+        file_name: str
```

Comment:
It says `file_name` here but the parameter passed is actually `fileName`

Reply:
I am using not fileName